### DISCUSSION TITLE
Add Ring of Dueling to Item Charges plugin (Notification).

### DISCUSSION THREAD

**Discussion by JonneSaloranta** [runelite/runelite#18234] [Ideas]
I've done quite a bit of activities as Ironman that require Ring of Dueling and have ran out of charges multiple times, so i was wondering if there would be any support for adding Ring of Dueling to the Item Charges plugin? I have already implemented and tested everything until i read the contribution guidelines (I know, rookie mistake.)

The implementation would include:

- Notification when the ring is destroyed (if notification is enabled)
- Uses Low and Very low warnings

**Comment by Alexsuperfly** [comment]
the warnings already work, you can use the chat notifications plugin to notify on the break message

**Reply by JonneSaloranta** [reply]
Valid point but most of the already included items with charges e.g ring of recoil could have also been implemented this way. Also i would rely on another plugin to do this.

**Reply by Alexsuperfly** [reply]
they predate the chat notifications plugin so they've been grandfathered in to being there, the chat notifications plugin works well for it otherwise

**Reply by JonneSaloranta** [reply]
So they don't approve any updates to the Item Charges plugin? In that case i might use the chat notifications plugin.

**Reply by Alexsuperfly** [reply]
additions can be made, its just that instead of waiting for something to be added you can just already start getting notifications now

**Reply by ShaunDreclin** [reply]
> the warnings already work, you can use the chat notifications plugin to notify on the break message

One problem with this, the chat notifications plugin does not let you set custom notification settings for different chat messages
